src/components/reviews/ReviewsOverview.js

```diff
@@ -67,11 +73,28 @@ const ReviewsOverview = (props) => {
       </div>
       <div className={classes.usersRecommend}>100% of reviews recommend this product</div>
       <div className={classes.breakdownContainer}>
-        <Breakdown title="5 stars" percentage={props.data.ratings[5] / total} />
-        <Breakdown title="4 stars" percentage={props.data.ratings[4] / total} />
-        <Breakdown title="3 stars" percentage={props.data.ratings[3] / total} />
-        <Breakdown title="2 stars" percentage={props.data.ratings[2] / total} />
-        <Breakdown title="1 stars" percentage={props.data.ratings[1] / total} />
+        <Breakdown title="5 stars" percentage={props.data.ratings[5] / total} onClick={() => props.toggleFilter(5)} />
+        <Breakdown title="4 stars" percentage={props.data.ratings[4] / total} onClick={() => props.toggleFilter(4)} />
+        <Breakdown title="3 stars" percentage={props.data.ratings[3] / total} onClick={() => props.toggleFilter(3)} />
+        <Breakdown title="2 stars" percentage={props.data.ratings[2] / total} onClick={() => props.toggleFilter(2)} />
+        <Breakdown title="1 stars" percentage={props.data.ratings[1] / total} onClick={() => props.toggleFilter(1)} />
```

Comment:
I missed that. What does the '/' do inside percentage?

Reply:
It uh, it divides.